src/api/memberships.js

```diff
@@ -0,0 +1,42 @@
+import * as postgrest from './postgrest';
+
+export const postMembershipReq = membership => {
+  return postgrest
+    .postJSON('/memberships', membership, { Prefer: 'return=representation' })
+    .then(membershipJSON => membershipJSON[0])
+    .catch(err => console.log('Error posting membership:', err));
+};
+
+export const getMembershipsReq = filterOption => {
+  const filterKey = Object.keys(filterOption)[0];
+  const filterVal = Object.values(filterOption)[0];
+  return postgrest
+    .getJSON('/memberships')
```

Comment:
Do ".getJSON(`/memberships?${filterKey}=eq.${filterVal}`)" and have the database do the filtering for you :-D

Reply:
Then you don't need to do the filtering below

Reply:
Nice! will do